### ISSUE TITLE
Functions with reserved words

### ISSUE THREAD

**Issue by Valdsonjr** [atom-haskell/language-haskell#52]
Function names defined as (reserved word)' are highlighted as reserved words (it seems that github has the same problem)
example:
``` Haskell
if' :: Bool -> a -> a -> a
if' True x  _ = x
if' False _ y = y

iff x y c d = if' ( x < y ) c d
```
note that iff is not highlighted but if' is

**Comment by lierdakil** [comment]
I'm aware of the problem, but it's not very easy to fix, since `'` is
considered word-boundary by underlying regexp engine. I might get away with
negative lookahead, but that can be quite fragile.

2015-09-20 18:44 GMT+03:00 Valdson <notifications@github.com>:

**Comment by lierdakil** [comment]
Okay, I've published 1.4.6, which hopefully doesn't break anything and
should detect described case satisfactory enough. Please confirm that it's
working out okay for you.

2015-09-20 22:16 GMT+03:00 Nikolay Yakimov <root@livid.pp.ru>:

**Comment by Valdsonjr** [comment]
Sorry for the delay,  It's working and I didn't find anything broken for now, I'll keep you updated.
